tools/Generator/Xamarin.Forms.Core.json

```diff
@@ -1660,7 +1660,16 @@
           "name": "Command",
           "defaultValue": "null",
           "inputType": "unit -> unit",
-          "convToModel": "makeCommand"
+          "convToModel": "makeCommand",
+          "attached": [
```

Comment:
This `attached` field is meant for a property that is a collection.
It allows to use attached properties on the children of this collection, like Grid.Column is only valid on the direct children of a Grid.
So here, it doesn't make sense to add the Accelerator there, as it's only valid on the MenuItem themselves.

There's 2 options to fix this:
- Add `Accelerator` as an attached property on the `Items` property of `Menu`
- Add `Accelerator` as a standard property on `MenuItem` but with a custom update logic that uses `MenuItem.SetAccelerator(menuItem, value)` instead.

I think the 2nd one is better, even if not quite exactly what XF does.